scripts/gerrit-handler.coffee

```diff
@@ -0,0 +1,74 @@
+# Handle Gerrit webhook notifications
```

Comment:
Part of this function looks very duplicated with redmine-handler, can't it be factorized?

Reply:
GerritNotifier and RedmineNotifier now are extended from NotifHandler class